File: customized-checkout/customers/dotnet/search_cards.cs
using mercadopago;
using System;
using System.Collections;

public class MyClass {
	public static void Main () {
		MP mp = new MP("ACCESS_TOKEN");

		Hashtable cards = mp.get ("/v1/customers/[CUSTOMER_ID]/cards");

		Console.WriteLine (cards["response"].ToString());
	}
}

File: customized-checkout/customers/java/search_cards.java
import com.mercadopago.MP;
import org.codehaus.jettison.json.JSONObject;
import org.codehaus.jettison.json.JSONArray;

public class MyClass {
	public static void main (String args[]) {
		MP mp = new MP ("ACCESS_TOKEN");

		JSONObject cards = mp.get ("/v1/customers/[CUSTOMER_ID]/cards");

		System.out.println(cards.getJSONObject("response").toString());
	}
}
